### PR TITLE
[python] added f-strings to python-package/setup.py

### PR DESCRIPTION
**Pull request by NovusEdge**

### CODE REVIEW

File: python-package/setup.py [hunk 1]
```diff
@@ -183,7 +183,7 @@ def compile_cpp(use_mingw=False, use_gpu=False, use_cuda=False, use_mpi=False,
                 arch = "Win32" if bit32 else "x64"
                 vs_versions = ("Visual Studio 16 2019", "Visual Studio 15 2017", "Visual Studio 14 2015")
                 for vs in vs_versions:
-                    logger.info("Starting to compile with %s (%s).", vs, arch)
+                    logger.info(f"Starting to compile with %s (%s).", vs, arch)
```
Comment (StrikerRUS) [inline python-package/setup.py:186]:
```suggestion
                    logger.info(f"Starting to compile with {vs} ({arch}).")
```

**Review by StrikerRUS — CHANGES_REQUESTED** [review]
One mistake below, everything else looks fine!

**Comment by NovusEdge** [comment]
> One mistake below, everything else looks fine!

Could you please mention the line where the mistake is.

**Comment by jameslamb** [comment]
> > One mistake below, everything else looks fine!
> 
> Could you please mention the line where the mistake is.

it is right here: https://github.com/microsoft/LightGBM/pull/4145#discussion_r604399154

**Comment by StrikerRUS** [comment]
@NovusEdge 
> Could you please mention the line where the mistake is.

GitHub review comments are posted with a small visible piece of code. Most often, the last line of that codepiece is the line review comment is referring to.

image


Also, GitHub has recently introduced "Review suggestions". So that reviewers now can suggest code changes directly in the review comments:

image

Contributors, in case they are agree with a suggestion, are able to commit the suggested change directly in the browser by clicking "Commit suggestion" button or commit multiple suggestion at one time ("Add suggestion to batch" button).

**Review by StrikerRUS — APPROVED** [review]
@NovusEdge Thanks for your contribution!

**Pull request merged**